generated/src/main/java/account/DefaultApi.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech) (6.6.0).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+package account;
+
+import account.Account;
+import account.Error;
+import account.NewAccount;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.Generated;
+
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
+public interface DefaultApi {
```

Comment:
You need to play with spec  tags and generate 2 controllers `AccountApi` and  `HealthcheckApi`